akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/jackson/ReplicatedEventSourcingJacksonSpec.scala

```diff
@@ -44,5 +45,9 @@ class ReplicatedEventSourcingJacksonSpec extends ScalaTestWithActorTestKit with
       val counter = WithCounter(Counter.empty)
       serializationTestkit.verifySerialization(counter)
     }
+    "serialize Tagged" in {
```

Comment:
I don't see how this has anything to do with Jackson?

Reply:
It has nothing to do with Jackson. Like caffeine deprivation. 🤦🏼‍♂️